### PR TITLE
Transaction.fromObject

### PR DESCRIPTION
**Pull request by fanatid**
Solve #18
Include #20 
@andrewoons @Levino @braydonf

### CODE REVIEW

**Comment by Levino** [comment]
Out of interest: How does it go from here? Is it good if I express my consent here? When and by whom are pull request accepted?

**Comment by fanatid** [comment]
@Levino I don't know exactly who accept pull requests, maybe @braydonf will be able to explain this process.. part of this theme was affected in https://github.com/bitpay/bitcore/issues/1252

File: lib/transaction/transaction.js [hunk 1]
```diff
@@ -374,15 +374,15 @@ Transaction.prototype.fromObject = function fromObject(arg) {
   _.each(transaction.outputs, function(output) {
     self.addOutput(new Output(output));
   });
+  if (transaction.fee) {
+    this.fee(transaction.fee);
```
Comment (braydonf) [inline lib/transaction/transaction.js]: Should we use `this._fee = transaction.fee` so that `_updateChangeOutput()` isn't called at all?

**Comment by braydonf** [comment]
Good find!

**Comment by fanatid** [comment]
@braydonf thanks for advice! Updated.

**Comment by braydonf** [comment]
ACK

**Pull request merged**